src/about/repl/repl/host/window.js

```diff
@@ -70,7 +70,7 @@ const evalContext =
 
 export const evaluate =
   (id/*:ID*/, code/*:string*/)/*:Task<Never, EvaluationResult>*/ =>
-  Task.future(() => new Promise(resolve => {
+  Task.create(resolve => {
```

Comment:
We need to keep a promise wrapper here in order to keep an async resolution behavior as it was prior to this change (See https://github.com/browserhtml/browserhtml/pull/1003#issuecomment-211648650 for more details)